v2/response.go

```diff
@@ -0,0 +1,56 @@
+package jsonrpc
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+// Error codes generated by a JSONRPC Server
+const (
+	CodeParseError        int = -32700
+	CodeInvalidRequest        = -32600
+	CodeMethodNotFound        = -32601
+	CodeInvalidParameters     = -32602
+	CodeInternalError         = -32603
+	CodeMiscError             = -32001
+)
+
+// Error will return any errors in the JSONRPC format.
+type Error struct {
```

Comment:
the only situation where this will cause an issue is where the package is imported into the file block without a qualifier.
Possible but not something that I would suggest you see that often. Go's namespacing is pretty good and stuttering is unnecessary.